Mathlib/Data/Finset/Lattice.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 
 ! This file was ported from Lean 3 source module data.finset.lattice
-! leanprover-community/mathlib commit 2d44d6823a96f9c79b7d1ab185918377be663424
+! leanprover-community/mathlib commit 1d29de43a5ba4662dd33b5cfeecfc2a27a5a8a29
```

Comment:
```suggestion
! leanprover-community/mathlib commit 2d44d6823a96f9c79b7d1ab185918377be663424
```
Dashboard says no need to update here? It's fine to update I guess, just flagging in case something is missing here.

Reply:
Yeah, this change was just plain wrong. I was hoping to check this when I woke up this morning, but I guess I'm too slow. @YaelDillies, can you make a PR to revert this line?

Reply:
What happened is that I accidentally partially reverted style changes to `data.finset.locally_finite` in leanprover-community/mathlib#18838, but in a way that only changes whitespace. So bumping the SHA was the right thing to do.